chia/harvester/harvester.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, root_path: Path, config: Dict, constants: ConsensusConstants)
         self.state_changed_callback: Optional[Callable] = None
         self.last_load_time: float = 0
         self.plot_load_frequency = config.get("plot_loading_frequency_seconds", 120)
+        self.parallel_read: bool = config.get("parallel_read", True)
```

Comment:
We should add the `parallel_read` field into `chia/util/initial-config.yaml`

Reply:
Added this